src/docs/pages/introduction/getting-started/react.md

```diff
@@ -109,6 +109,107 @@ There are a few cases where native events of Web Components do not behave as exp
+- [LdBreadcrumbs](/components/ld-breadcrumbs/) component is a simple way to display the current location in your application.
+
+As this recipe focuses on how to put together Liquid Oxygen and React Router, we'll only highlight the relevant parts of the code. We assume you already have a basic React Router setup in place. If not, please refer to the [React Router documentation](https://reactrouter.com/en/start/tutorial).
+
+Let's get started with routing. We cannot use the `<Link>` component provided by React Router within LdSidenavNavitem, therefore we use the [useNavigate](https://reactrouter.com/en/main/hooks/use-navigate) hook to navigate to a different route.
+
+```tsx
+// Sidebar.tsx
+//...
+const navigate = useNavigate()
+
+const handleLink = useCallback(
+  (e: React.MouseEvent) => {
+    e.preventDefault()
+    const href = (e.target as HTMLLdSidenavNavitemElement).href
+    if (href) {
+      navigate(href)
+    }
+  },
+  [navigate]
+)
+
+return (
+  // ...
+  <LdSidenavNavitem
+    href={`/mathematical-foundations/discrete-mathematics`}
+    onClick={handleLink}
+  >
+    Discrete Mathematics
+  </LdSidenavNavitem>
+  // ...
+)
+// ...
+```
+
+The `href` prop of `LdSidenavNavitem` tells the component to add `LdLink` which renders a proper anchor tag. This is important for accessibility. As this would already work to navigate but bypass the client-side navigation of React Router, we also add a click handler. The click handler uses the `navigate` function from the `useNavigate` hook to navigate to the specified href. `preventDefault()` is called to prevent the default behavior of the anchor tag.
+
+`LdSidenav` provides visual indicators for an active item. We can use the `pathname` property of the `useLocation()` hook provided by React Router to determine the active route and set `selected` accordingly.
+
+```tsx
+// Sidebar.tsx
+//...
+const { pathname } = useLocation()
+
+return (
+  // ...
+  <LdSidenavNavitem
+    href={`/mathematical-foundations/discrete-mathematics`}
+    onClick={handleLink}
+    selected={pathname === '/mathematical-foundations/discrete-mathematics'} // Depending on your routes, you'll need a more sophisicated evaluation here
+  >
+    Discrete Mathematics
+  </LdSidenavNavitem>
+  // ...
+)
+// ...
+```
+
+Using the same hooks, you can generate breadcrumbs from `pathname`. It's on you to resolve the path to human readable lables.
+
+```tsx
+// Breadcrumbs.tsx
+// ...
+const navigate = useNavigate()
+const { pathname } = useLocation()
+const crumbs = useResolvedCrumbs(pathname) // This is up to you 😉
+
+const handleLink = useCallback(
+  (e: React.MouseEvent) => {
+    e.preventDefault()
+    const href = (e.target as HTMLLdCrumbElement).href
+    if (href) {
+      navigate(href)
+    }
+  },
+  [navigate]
+)
+
+return (
+  <LdBreadcrumbs>
+    {crumbs.map((crumb) => (
+      <LdCrumb key={crumb.label} onClick={handleLink} href={crumb.href}>
```

Comment:
Is the `key` prop necessary here? If so, maybe we should explain, why.

Reply:
Whenever I forget it i get warnings in the browser that react needs the key when iterating over the elements. The example should not come with warnings so i added it. 

Contrary to the examples above I use a map here to introduce the idea of how to get from `location` to `breadcrumbs` which I think would be dull to show some static coded stuff.